src/services/api/types.ts

```diff
@@ -0,0 +1,33 @@
+export interface authResponse {
+  accessToken: string;
+  refreshToken: string;
+  user: IUser;
+}
+
+export interface IUser {
+  id: number;
+  firstName: string;
+  lastName: string;
+  email: string;
+}
+
+export interface IProduct {
+  id: number;
+  name: string;
+  description: string;
+  price: number;
+  currency: string;
+  img: string;
+}
+
+export interface IOrder {
+  id: number;
+  userId: number;
+  productId: number;
+}
+
+export interface IDecodedUser {
+  data: IUser;
```

Comment:
I would not store `firstName`, `lastName` and `email` in the token, since this data usually may be changed by user in his account. For this project - ok.